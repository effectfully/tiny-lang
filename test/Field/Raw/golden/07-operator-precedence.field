ext a;
ext b;

assert
(neg a and b) <= (neg (a and b))
;
