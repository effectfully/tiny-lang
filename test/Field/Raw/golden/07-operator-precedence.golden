Program [(Var "a",BuiltIn (Some Field)),(Var "b",BuiltIn (Some Field))] Statements [EAssert (EAppBinOp FLe (EAppBinOp And (EAppUnOp Neg (EVar (Var "a"))) (EVar (Var "b"))) (EAppUnOp Neg (EAppBinOp And (EVar (Var "a")) (EVar (Var "b")))))]